webofneeds/won-matcher/src/main/java/won/matcher/component/MatcherNodeURISource.java

```diff
@@ -1,31 +0,0 @@
-/*
- * Copyright 2012  Research Studios Austria Forschungsges.m.b.H.
- *
- *    Licensed under the Apache License, Version 2.0 (the "License");
- *    you may not use this file except in compliance with the License.
- *    You may obtain a copy of the License at
- *
- *        http://www.apache.org/licenses/LICENSE-2.0
- *
- *    Unless required by applicable law or agreed to in writing, software
- *    distributed under the License is distributed on an "AS IS" BASIS,
- *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- *    See the License for the specific language governing permissions and
- *    limitations under the License.
- */
-
-package won.matcher.component;
-
-import java.util.Iterator;
-
-/**
- * Interface that is used to obtain a WON_BA node URI.
- */
-public interface MatcherNodeURISource
```

Comment:
We cannot delete this interface and its implementations: they are still used. I am sceptical if we should really delete won-matcher at all.. The bots will not work without them.